cmd/kubeadm/app/componentconfigs/defaults.go

```diff
@@ -33,6 +34,25 @@ import (
 const (
 	// KubeproxyKubeConfigFileName defines the file name for the kube-proxy's kubeconfig file
 	KubeproxyKubeConfigFileName = "/var/lib/kube-proxy/kubeconfig.conf"
+
+	// KubeletReadOnlyPort specifies the default insecure http server port
+	// 0 will disable insecure http server.
+	KubeletReadOnlyPort int32 = 0
```

Comment:
Let's make the consts package private. Under normal circumstances, they should not be used outside of this package.

Reply:
Thanks a lot, fixed.